splunk/resource_splunk_admin_saml_groups.go

```diff
@@ -130,21 +132,29 @@ func getAdminSAMLGroupsConfig(d *schema.ResourceData) (adminSAMLGroupsObject *mo
 
 func getAdminSAMLGroupsByName(name string, httpResponse *http.Response) (AdminSAMLGroupsEntry *models.AdminSAMLGroupsEntry, err error) {
 	response := &models.AdminSAMLGroupsResponse{}
+	_ = json.NewDecoder(httpResponse.Body).Decode(response)
```

Comment:
This was already being done for every `httpResponse.StatusCode` value, so I moved it out of the `switch` scope.